runners/spark/src/main/java/org/apache/beam/runners/spark/translation/streaming/UnboundedDataset.java

```diff
@@ -81,12 +81,16 @@ public UnboundedDataset(Iterable<Iterable<T>> values, JavaStreamingContext jssc,
     return dStream;
   }
 
-  @Override
   public void cache() {
     dStream.cache();
   }
 
   @Override
+  public void cache(String storageLevel) {
+    // ignore: we "force" MEMORY storage level in streaming
```

Comment:
This will cause no-op although we may have explicitly called `cache(storageLevel)`, I think we should choose:
1. throw `UnsupportedOperationException` to fail if someone tried to call `cache(storageLevel)`
OR
2. WARN in log and call `cache()` to avoid failure and "force" as storage level.

I'm fine with (2) since there's no way for a user to "fix" it, only one storage level would be fine.